applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -416,6 +545,7 @@ static void uart_rx_handler(u8_t character)
 static void uart_callback(struct uart_event *evt, void *user_data)
 {
 	int err;
+	static u16_t pos;
```

Comment:
Use one static pointer for multiple buffers is not recommended

Reply:
Only 1 buffer is reported back to user at a time, so one pointer should be enough.